src/Components/WebView/WebView/src/Services/WebViewErrorBoundaryLogger.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Components.Web;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Components.WebView.Services;
+
+internal sealed class WebViewErrorBoundaryLogger : IErrorBoundaryLogger
+{
+    private readonly ILogger<ErrorBoundary> _errorBoundaryLogger;
+
+    public WebViewErrorBoundaryLogger(ILogger<ErrorBoundary> errorBoundaryLogger)
+    {
+        _errorBoundaryLogger = errorBoundaryLogger;
+    }
+
+    public ValueTask LogErrorAsync(Exception exception)
+    {
+        // For, client-side code, all internal state is visible to the end user. We can just
+        // log directly to the console.
+        _errorBoundaryLogger.LogError(exception, exception.ToString());
```

Comment:
```suggestion
        _errorBoundaryLogger.LogError(exception, exception.Message);
```
I'm not sure if I'm right, but most `ILogger` implementations would add `exception.ToString()` to their log and passing `exception.ToString()` might be redundant.
https://github.com/dotnet/aspnetcore/blob/5fb8db4fdb67ca22f18f356df4916dceccd05066/src/Components/WebAssembly/WebAssembly/src/Services/WebAssemblyConsoleLogger.cs#L136